spec/unit/webdriver_proxy_spec.ts

```diff
@@ -0,0 +1,118 @@
+import * as nock from 'nock';
+
+import {CommandName, WebDriverCommand} from '../../lib/webdriver_commands';
+import {WebDriverProxy} from '../../lib/webdriver_proxy';
+
+import {InMemoryReader, InMemoryWriter, TestBarrier} from './util';
+
+describe('WebDriver Proxy', () => {
+  let proxy: WebDriverProxy;
+
+  beforeEach(() => {
+    proxy = new WebDriverProxy(`http://localhost:4444/wd/hub`);
```

Comment:
This is a nonsense url, right? Can we make that clearer, like `http://test_webdriver_url`?